[V2-3568] adding a blog to your store (#46)
volusion/element-documentation: explanations/adding-a-blog/README.md

# Adding a Blog to Your Store

> Generally speaking when you want to incorporate a blog into your store, you will have to redirect your users to other domain (or subdomain) like `blog.mystore.com`.

Reviewer: ```suggestion
Generally speaking when you want to incorporate a blog into your store, you will have to redirect your users to some other domain (or subdomain) like `blog.mystore.com`.
```

That means that your blog is hosted in a totally different platform and you have to invest time and money creating a similar look and feel for your blog.

If you have invested in creating a nice header and footer for your store for example, you will have to create it again possibly in a totally different platform. Not to mention
that you now need to care about your store's hosting and your blog's hosting.

> Also, if you want your users to use the cart from your blog, all you can do is a hard redirect. As a result of that, you can't have a first class experience from your blog

Reviewer 2: ```suggestion
Also, if you want your users to use the cart from your blog, all you can do is link to a product details page, forcing a full page reload. As a result of that, you can't have a first class experience from your blog
```

Reviewer 2: (just a thought)

Author: @divinentd I was just thinking on opening the cart, not even adding products. I have seen some blogs redirecting to the cart contents as they can't open the cart in the blog. Can you think a better way to express that?

Reviewer 2: The core issue regardless of viewing product or adding to cart is you have to do a full page reload, because your blog and shopping cart aren't part of the same single page app.

maybe, "Also, if you want your users to use the cart from your blog shoppers will have to be linked to a new page."

to add products to the cart or check cart details without abandoning the blog page.

Additionally, you are sacrificing SEO value by not having your blog hosted as a route in your main domain (`www.yourstore.com/blog`)

But with Element you can do avoid all that. By creating a `/blog` page to your store and adding the appropriate blocks, you will be able to offer a first class
experience to your users while reading your content. As a result of that, you can have another point to increase sales by embedding add to cart operations from
your blog posts or any kind of actions that users can normally do from the store. Also you are reusing the same store's theme, reducing maintenance costs as

> exposed before.

Reviewer 2: You could end this sentence at "reducing maintenance."

> To see how easy you can add a blog, read our [tutorial](/tutorials/building-an-element-page/README.md) where all the steps are described from start to finish.

Reviewer 2: ```suggestion
To see how easy you can add a blog, read our [Building an Element Page Tutorial](/tutorials/building-an-element-page/README.md) where all the steps are described from start to finish.
```

> To see how easy you can add a blog, read our [tutorial](/tutorials/building-an-element-page/README.md) where all the steps are described from start to finish.

Reviewer: ```suggestion
To see how easily you can add a blog, read our [Building an Element Page Tutorial](/tutorials/building-an-element-page/README.md) where all the steps are described from start to finish
```
@divinentd added "easily"
